master/reference/cni-plugin/configuration.md

```diff
@@ -76,6 +76,12 @@ When using Calico IPAM, the following flags determine what IP addresses should b
 
 A specific IP address can be chosen by using [`CNI_ARGS`](https://github.com/appc/cni/blob/master/SPEC.md#parameters) and setting `IP` to the desired value.
 
+The list of possible IPv4 and IPv6 pools can also be specified via the following properties:
```

Comment:
I think you need a newline after this otherwise the bullets won't format properly.